addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+account      / account.reconcile.model.template / match_amount (selection)      : NEW selection_keys: ['between', 'greater', 'lower']
+account      / account.reconcile.model.template / match_amount_max (float)      : NEW 
+account      / account.reconcile.model.template / match_amount_min (float)      : NEW 
+account      / account.reconcile.model.template / match_journal_ids (many2many) : NEW relation: account.journal
+account      / account.reconcile.model.template / match_label (selection)       : NEW selection_keys: ['contains', 'match_regex', 'not_contains']
+account      / account.reconcile.model.template / match_label_param (char)      : NEW 
+account      / account.reconcile.model.template / match_nature (selection)      : NEW required: required, selection_keys: ['amount_paid', 'amount_received', 'both'], req_default: function
+account      / account.reconcile.model.template / match_partner (boolean)       : NEW 
+account      / account.reconcile.model.template / match_partner_category_ids (many2many): NEW relation: res.partner.category
+account      / account.reconcile.model.template / match_partner_ids (many2many) : NEW relation: res.partner
+account      / account.reconcile.model.template / match_same_currency (boolean) : NEW 
+account      / account.reconcile.model.template / match_total_amount (boolean)  : NEW 
+account      / account.reconcile.model.template / match_total_amount_param (float): NEW 
+account      / account.reconcile.model.template / rule_type (selection)         : NEW required: required, selection_keys: ['invoice_matching', 'writeoff_button', 'writeoff_suggestion'], req_default: function
+???
+
+account      / account.tax              / cash_basis_account (many2one) : was renamed to cash_basis_account_id [nothing to do]
+# DONE: pre-migration: renamed
+
+account      / account.tax              / cash_basis_base_account_id (many2one): previously in module account_cash_basis_base_account
+# NOTHING TO DO
+
+account      / account.tax              / tax_adjustment (boolean)      : DEL
+account      / account.tax              / type_tax_use (selection)      : selection_keys is now '['adjustment', 'none', 'purchase', 'sale']' ('['none', 'purchase', 'sale']')
+# DONE: post-migration: added the boolean field as a new key in the selection
+
+account      / account.tax.template     / cash_basis_account (many2one) : was renamed to cash_basis_account_id [nothing to do]
+# DONE: pre-migration: renamed
+
+account      / account.tax.template     / cash_basis_base_account_id (many2one): NEW relation: account.account.template
+account      / account.tax.template     / company_id (many2one)         : DEL relation: res.company
+# NOTHING TO DO
+
+account      / account.tax.template     / tax_adjustment (boolean)      : DEL
+account      / account.tax.template     / type_tax_use (selection)      : selection_keys is now '['adjustment', 'none', 'purchase', 'sale']' ('['none', 'purchase', 'sale']')
```

Comment:
Group together these lines and the ones in `account.tax` for better comprehension.